cmd/ak/cmd/runtimes/runtimes.go

```diff
@@ -3,9 +3,20 @@ package runtimes
 import (
 	"github.com/spf13/cobra"
 
+	"go.autokitteh.dev/autokitteh/backend/runtimes"
 	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
 )
 
+var local bool
+
+func client() sdkservices.Runtimes {
```

Comment:
See other parent commands - the convention there is to add a function with the name of the client, i.e. `func runtimes()` instead of `func client()`. Also, please place this helper function at the bottom to emphasize that `runtimesCmd` is the focus of this file.